packages/authjs-nuxt/src/runtime/composables/useAuth.ts

```diff
@@ -50,6 +50,16 @@ export function useAuth() {
     }
   }
 
+  const getProviders: () => ReturnType<typeof auth.getProviders> = () => {
+    try {
+      return auth.getProviders()
+    }
+    catch (error) {
+      status.value = "error"
```

Comment:
Should we remove the trycatch here. ? If this error, the session status shouldn't be affected.

Reply:
You are correct